client/src/scenes/Home/SyncRequestForm/SyncRequestForm.tsx

```diff
@@ -36,7 +36,7 @@ export const SyncRequestForm = () => {
             <GoogleCalendarPicker />
             <CoursesPicker onChange={setCourses} />
             <StudentGroupPicker onChange={setStudentGroups} />
-            <ColorPicker onChange={setColorId} />
+            <ColorPicker onChange={setColorId} defaultColor={calendar?.backgroundColor} />
```

Comment:
The default color can be managed and abstract directly in the ColorPicker

Use the calendar selector inside it to retrieve the calendar color